### PR TITLE
gh-104876: Remove deprecated turtle.RawTurtle.settiltangle

### LINKED ISSUE #104876: Remove deprecated turtle.RawTurtle.settiltangle method
# Feature or enhancement

The `turtle.RawTurtle.settiltangle` method was deprecated in docs since Python 3.1 and with a warning since 3.11, and scheduled for removal in 3.13.

# Previous discussion

* https://github.com/python/cpython/pull/29618

### Linked PRs
* gh-104877

### PR DESCRIPTION
**Pull request by hugovk**
Closes #104876.

Remove the `turtle.RawTurtle.settiltangle` method, deprecated in docs since Python 3.1 and with a warning since Python 3.11.

----
:books: Documentation preview :books:: https://cpython-previews--104877.org.readthedocs.build/

* Issue: gh-104876

### CODE REVIEW

File: Lib/turtle.py [hunk 1]
```diff
@@ -2935,8 +2908,8 @@ def tiltangle(self, angle=None):
         between the orientation of the turtleshape and the heading of the
         turtle (its direction of movement).
 
-        (Incorrectly marked as deprecated since Python 3.1, it is really
-        settiltangle that is deprecated.)
+        (Incorrectly marked as deprecated since Python 3.1, it was really
+        settiltangle that was deprecated (and removed in 3.13).)
```
Comment (vstinner) [inline Lib/turtle.py]: I don't get the purpose of this remark. Why not just removing this part? It's misleading to have a comment about an old mistake, now fixed.

File: Doc/whatsnew/3.13.rst [hunk 2]
```diff
@@ -118,6 +118,11 @@ Removed
 * Remove support for using :class:`pathlib.Path` objects as context managers.
   This functionality was deprecated and made a no-op in Python 3.9.
 
+* Remove the :meth:`!turtle.RawTurtle.settiltangle` method,
+  deprecated in docs since Python 3.1
+  and with a warning since Python 3.11.
```
Comment (vstinner) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  and with a deprecation warning since Python 3.11.
```

same change for the Changelog entry.

**Review by vstinner — APPROVED** [review]
LGTM, but I left minor remarks. Also, I never used the turtle module, so I don't know how to test this change. ``Lib/test/test_turtle.py`` checks that there is no regression, no?

File: Lib/turtle.py (hunk 1 above)
Comment (hugovk) [reply Lib/turtle.py]: Good point, will remove it completely now.

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**